tools/mo/unit_tests/mo/front/tf/loader_test.py

```diff
@@ -3,19 +3,15 @@
 
 import unittest.mock
 from io import StringIO
-from unittest.mock import Mock, MagicMock
-
-from generator import generate, generator
-
 from openvino.tools.mo.front.tf.loader import load_tf_graph_def
+from unittest.mock import Mock, MagicMock
 
 
-@generator
 class TestLoader(unittest.TestCase):
-    @generate('/path/to/somewhere/my_checkpoint.ckpt', '/path/to/somewhere/my_meta_graph.meta')
     @unittest.mock.patch('sys.stdout', new_callable=StringIO)
-    def test_helper_print_ckpt(self, path, out):
-        mock = Mock(__bool__=MagicMock(side_effect=Exception()))
-        self.assertRaises(Exception, load_tf_graph_def, path, meta_graph_file=mock)
-        self.assertRegex(out.getvalue(),
-                         r'\[ WARNING ] The value for the --input_model command line parameter ends with "\.ckpt"')
+    def test_helper_print_ckpt(self, out):
```

Comment:
can we use parametrized pytest tests https://docs.pytest.org/en/latest/example/parametrize.html ?

Reply:
These are legacy tests for `openvino-dev` component so I decided not to waste time and apply simple solution.
pytest parametrization can conflict with unittest for some case and I want to avoid it.